config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -375,7 +375,7 @@ periodics:
   extra_refs:
   - org: kubernetes-sigs
     repo: cluster-api-provider-azure
-    base_ref: ${capz_periodic_branch_name}
+    base_ref: main # TODO: switch back to capz_periodic_branch_name once capz release-1.12 exists
```

Comment:
alternative would be to backport the PR but since it's quite large and touches template this is the quickest way to get signal. release-1.12 should be available mid November, at which point we can revert this change.